app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
need to add this relation in the user model too. 

`User`
`has_many :comments, dependent: nullify`